src/api/handlers/guild.ts

```diff
@@ -455,8 +455,8 @@ export async function getAuditLogs(
   });
 }
 
-/** Returns the guild embed object. Requires the MANAGE_GUILD permission. */
-export async function getEmbed(guildID: string) {
+/** Returns the guild widget object. Requires the MANAGE_GUILD permission. */
+export async function getWidgetSettings(guildID: string) {
```

Comment:
Better name? How about just `getWidget`?

Reply:
This Endpoint returns the settings for the Widget. There is an other Endpoint hat returns the Widget so naming this one getWidget makes more sense to me.